### PR TITLE
shebang fix, along with nixos files for easy environment creation on nix

### PR DESCRIPTION
**Pull request by amanjeev**
low priority.

### CODE REVIEW

File: .envrc
Comment (jonathanpallant) [inline .envrc:1]: This is https://direnv.net, right?

File: .envrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# the shebang is ignored, but nice for editors
+
+if type -P lorri &>/dev/null; then
+  eval "$(lorri direnv)"
```
Comment (jonathanpallant) [inline .envrc:5]: what is `Lorri`?

File: .envrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+else
+  echo 'while direnv evaluated .envrc, could not find the command "lorri" [https://github.com/nix-community/lorri]'
```
Comment (jonathanpallant) [inline .envrc:7]: That seems noisy for anyone using direnv but not nix or lorri

**Comment by amanjeev** [comment]
yea will just fix shebang.

**Pull request closed without merging**